.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I would complain about  `.eslintrc.js` and not json, but lgtm :see_no_evil:

Reply:
Just wanted to prevent writing any `foo = require()` code in 2023.

Reply:
(new eslint config is [`eslint.config.js`](https://eslint.org/docs/latest/use/configure/configuration-files-new) :wink: )

Reply:
I know and I am happy when it is finally in stage 4: https://github.com/eslint/eslint/issues/13481